ha/monitor/k8s/parser.py

```diff
@@ -25,10 +25,15 @@
 from ha.monitor.k8s.alert import K8sAlert
 
 from cortx.utils.log import Log
+from ha.util.health_event_schema import Event
```

Comment:
This will change after class in utils gets merged.

Reply:
Yes.